packages/ui-alerts/src/Alert/__examples__/Alert.examples.ts

```diff
@@ -28,22 +28,17 @@ export default {
     children: [
       'An alert with some content',
       'an alert with a ton of content that is going to wrap. It takes a ton of content to ' +
-      'get text to wrap when the screen is high resolution and there is a good amount of space ' +
-      'to fill'
+        'get text to wrap when the screen is high resolution and there is a good amount of space ' +
+        'to fill'
     ]
   },
-  // @ts-expect-error ts-migrate(6133) FIXME: 'props' is declared but its value is never read.
-  getComponentProps: (props) => {
+  getComponentProps: (_props: any) => {
     return {
       margin: 'medium',
       screenReaderOnly: false,
       liveRegionPoliteness: 'polite',
       transition: 'none',
       open: true
     }
-  },
-  // @ts-expect-error ts-migrate(6133) FIXME: 'props' is declared but its value is never read.
-  filter: (props) => {
-    return props.screenReaderOnly || props.open === false
```

Comment:
If something is defined in getComponentProps, it can have that single value only